samples/powercli/job.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: vsphere-script
+  labels:
+    # Make this Job match the binding!
+    role: vsphere-job
+spec:
+  template:
+    metadata:
+      labels:
+        # So it's easier to list the Pods
+        role: vsphere-job
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: dump-events
+        image: docker.io/vmware/powerclicore
+        command: ["pwsh", "-Command"]
+        args:
+        - |
+          # Log into the VI Server
+          Set-PowerCLIConfiguration -InvalidCertificateAction Ignore -Confirm:$false | Out-Null
+          Connect-VIServer -Server ([System.Uri]$env:GOVC_URL).Host -User $env:GOVC_USERNAME -Password $env:GOVC_PASSWORD
```

Comment:
could the problem with the sim not working be that you don't have the GOVC_INSURE env variable specified here?
GOVC_INSECURE=1
I assume the -InvalidCertificateAction ignore would do the same, but just curious if you tried that as well?

Reply:
Yeah, I don't have this keying off of `GOVC_INSECURE`, but the line above is what that *should* guard.